Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Build the manager binary
-ARG GOLANG_VERSION=golang:1.19.3
```

Comment:
Looks like 1.19.8 is the last version here :-(

Reply:
docker.io got golang image for 1.19.12 :-) perhaps we could wait for 1 or 2 days for 1.19.13

Reply:
I would say let's wait a few days if it shows up. If it doesn't we can start asking around

Reply:
Ah, my bad. did wrongly look it up 👍

Reply:
xref https://github.com/docker-library/golang/issues/488